liquidator/OneInchExchange.js

```diff
@@ -24,20 +22,53 @@ class OneInchExchange {
     this.oneSplitContract = new web3.eth.Contract(OneSplit.abi, this.oneSplitAddress);
 
     this.toBN = web3.utils.toBN;
+
+    // 1 Split config
+    this.oneInchFlags = 0; // Enables all exchanges
+
+    // Number of pieces source volume could be splitted
+    // (Works like granularity, higly affects gas usage.
+    // Should be called offchain, but could be called onchain
+    // if user swaps not his own funds, but this is still considered
+    // as not safe)
+    // More info: https://github.com/CryptoManiacsZone/1inchProtocol#getexpectedreturn
+    this.oneInchParts = 2;
+  }
+
+  /**
+   * @notice Gets expected returns on one inch
+   * @param {Object} swapArgs - Swap arguments
+   * @param {string} swapArgs.fromToken Address of token to swap from
+   * @param {string} swapArgs.toToken Address of token to swap to.
+   * @param {string} swapArgs.amountWei String amount to swap, in Wei.
+   * @param {Object} options Web3 options to supply to send, e.g.
+   *      { from: '0x0...',
+            value: '1000',
+            gasPrice: '... }
+   */
+  async getExpectedReturn({ fromToken, toToken, amountWei }) {
```

Comment:
I like how you are wrapping parameters into the `swapArgs` object; it's pretty clean and makes it very clear what this is and the objects being passed in. 

My only issue is this is completely different to the rest of the bot and our whole repo. We don't pass variables like this anywhere else. Prehaps we should change the bots to deal with parameters like this. For example, the `index` of the [monitor](https://github.com/UMAprotocol/protocol/blob/dccb2feb9e5aa883648ee7dab4896c1bc6d92694/monitors/index.js#L43) has a bunch of parameters. if would be cleaner to pass them in as an object, but right now we don't do it like that.

My point is that even if there is a better pattern (and I think yours here is) it should still be consistent with the rest of the repo.

Reply:
This has been merged into the repo, it'll be extra work to remove this and then re-edit it later on down the line. Do you still want to change it now?

Reply:
this function `getExpectedReturn` has not been merged in though so by adding this is inconsistent. It's fine for you to leave it for now as long as we get the #1869 in soon after this PR is merged.